spec/controllers/link_check_reports_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "spec_helper"
+
+describe LinkCheckReportsController, type: :controller do
+  describe "#create" do
+    let(:service) { double(:service, call: nil) }
```

Comment:
Can we use the real service here?

Reply:
We've mirrored the other controller specs. I'd rather leave this inline with how the rest of the application tests are structured. That said, I'd probably agree that using the real thing would be preferable. Perhaps an issue for another time?

Reply:
My concern here is that I can see some of the code was using fields like `link_reportable_type` which no longer exists, if this was a real object, the tests would have caught it.

Reply:
Especially as the controllers feel like one level above from the model unit tests (and arguably they're more like integration tests) it seems like the controllers should be using the real models.

Reply:
Totally agree though Tasmin has now addressed those comments.

Is there a place we can put these kinds of technical debt issues? It feels like we should update all the controller specs in this repo.

Reply:
Probably good enough as a GitHub issue, although since we're currently working in these controller tests I still reckon it's worth updating these ones in this PR.

Reply:
@tsteer ^^